pkgs/tools/security/pbis/default.nix

```diff
@@ -0,0 +1,70 @@
+{ stdenv, fetchFromGitHub, autoconf, automake, libtool, perl, flex, bison, curl,
+  pam, popt, libiconv, libuuid, openssl_1_0_2, cyrus_sasl, sqlite, tdb, libxml2 }:
+
+stdenv.mkDerivation rec {
+  pname = "pbis-open";
+  version = "9.1.0";
+
+  src = fetchFromGitHub {
+    owner = "BeyondTrust";
+    repo = pname;
+    rev = version;
+    sha256 = "081jm34sf488nwz5wzs55d6rxx3sv566x6p4h1yqcjaw36174m8v";
+  };
+
+  nativeBuildInputs = [
+    autoconf automake libtool perl flex bison
+  ];
```

Comment:
to fix darwin build
```suggestion
  ] 
  # needs gencat
  ++ stdenv.lib.optionals stdenv.isDarwin [ glibc ];
```

Reply:
`glibc` has `meta.platform = lib.platforms.linux;`. I tried to add darwin to it and to build with `withLinuxHeaders = false;`, but it failed for some reason...

Reply:
oh, I'm sure there's a way to get gencat available, I'm just not sure how